gpii/node_modules/solutionsRegistry/src/schemas/solution-schema.json5

```diff
@@ -51,7 +51,11 @@
                             "type": "object",
                             "additionalProperties": false,
                             "properties": {
-                                 "schema": { "$ref": "gss-v7-full#", "required": true }
+                                 "schema": { "$ref": "gss-v7-full#", "required": true },
+                                 "path": {
```

Comment:
This seems like an inappropriate place to put this.  As it stands you're basically saying that any setting might have a path, and that it has to be either a string or object.   The individual settings should define the fact that they use the path.  

I'd also argue that you could be a lot more specific than just accepting an object.  I assume we know what properties such an object would have, i. e. we can only accept `get` and `set`, and it must have both of them to be a usable object.  I realise that this results in a lot of repeated boilerplate, but this will go away with the LSR work, where we can have a grade for all SPI settings where we express this once.